src/components/tables/CollectionsTable.vue

```diff
@@ -2,30 +2,31 @@
   <table class="table table-condensed table-responsive">
     <thead>
       <tr>
-        <th scope="col" class="pr-2"></th>
-        <th scope="col">Collection</th>
-        <th scope="col">Type</th>
-        <th scope="col">Materials</th>
-        <th scope="col">Standards</th>
-        <th scope="col">#Samples</th>
+        <th class="col-3" scope="col">Collection</th>
+        <th class="col-2" scope="col">Type</th>
+        <th class="col-2" scope="col">Materials</th>
+        <th class="col-2" scope="col">Standards</th>
+        <th class="col-2" scope="col">#Samples</th>
+        <th class="col-1" scope="col"></th>
       </tr>
     </thead>
     <tbody>
       <template v-for="(collection, index) in topLevelElements">
         <tr :key="index">
-          <td class="pr-1">
+          <!-- <td class="pr-1">
             <collection-selector
               class="collection-selection-button"
               :collectionData="collection"
               icon-only
               bookmark></collection-selector>
-          </td>
+          </td> -->
           <td
             :class="{
-              'table-text-content-columns-has-sub': hasSubCollections(
-                collection
-              ),
+              'table-text-content-columns-has-sub': hasSubCollections(collection),
               'table-text-content-columns': !hasSubCollections(collection),
+              'col-3': column === 'name',
```

Comment:
better turn this into a method with a switch statement for readability

Reply:
I created isWideColumn function similar to hasSubCollections